ldaptor/protocols/pureber.py

```diff
@@ -363,8 +363,7 @@ def __repr__(self):
         for tag, class_ in self.Identities.items():
             identities.append('0x{:02x}: {}'.format(tag, class_.__name__))
 
-        if sys.version_info < (3, 6):
-            identities.sort()
+        sys.version_info < (3, 6) and identities.sort()
```

Comment:
this is fine, but as a general rule I am -1 for making changes just to please a tool.

The code should be easy to read and the coverage tools should be there to help identify possible errors.

The previous line should be left as it is as codecov should be able to merge the coverage from python 3.5 and 3.8.

When you run the local `tox -e py38-test-dev` this line will be highlighted, but that should be ok.

Reply:
@adiroiban sorry I don't understand if you want me to rever this change or leave it

Reply:
You  can leave it...  I hope that python 3.5 support will be removed soon :)

My comment is that we should not put extra work with the single purpose of pleasing a tool.
This is a sign that the tool failed us :)